src/Microsoft.Health.Dicom.Core/Serialization/ConfigurationJsonConverter.cs

```diff
@@ -0,0 +1,81 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Health.Dicom.Core.Serialization;
+
+internal sealed class ConfigurationJsonConverter : JsonConverter<IConfiguration>
```

Comment:
I think it's quite creative to utilize `IConfiguration` to represent any customer input. 
I'm not expert on `IConfiguration`, not sure if there is any potential problems. 
I'll give an approve on this PR as it's under feature flag.